staging/src/k8s.io/kms/internal/plugins/mock/go.mod

```diff
@@ -3,7 +3,7 @@ module k8s.io/kms/plugins/mock
 go 1.19
 
 require (
-	k8s.io/klog/v2 v2.90.0
+	k8s.io/klog/v2 v2.90.1
```

Comment:
Ran `go mod tidy` to fix e2e job failure:
> CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} GO111MODULE=on go build -a -o mock-kms-plugin plugin.go
go: downloading k8s.io/klog/v2 v2.90.0
go: updates to go.mod needed; to update it:
	go mod tidy